### PR TITLE
Get contracts compiling / tested with truffle

### PR DESCRIPTION
**Pull request by gnidan**
(In progress, do not merge)

Might need to change the host in `truffle.js`, as well as the OAR value in `contracts/ExchangeRate.sol`

I removed all the copy-pasted contracts because they're still in master and we can find them there.

This assumes you have testrpc installed / running, and have done `npm install`. This also requires using `ethereum-bridge`, which connects to the testrpc node and handles the Oraclize requests. (https://github.com/oraclize/ethereum-bridge)

Basically, I do:

1. Run `testrpc`
2. Run `cd ethereum-bridge && node bridge -H 0.0.0.0:8545 -a 9` (selecting the 9th testrpc account, which we won't use)
3. Wait for the ethereum-bridge node to finish running and copy/paste the line that says `OAR = OraclizeResolverAddrI(...)` and copy that in place of the line in ExchangeRate.sol
4. Run `truffle test`

### CODE REVIEW

**Comment by gnidan** [comment]
I'm merging this @CyberNano
